Mathlib/Analysis/Normed/Group/AddTorsor.lean

```diff
@@ -145,35 +145,38 @@ theorem dist_vsub_vsub_le (p₁ p₂ p₃ p₄ : P) :
 
 theorem nndist_vadd_vadd_le (v v' : V) (p p' : P) :
     nndist (v +ᵥ p) (v' +ᵥ p') ≤ nndist v v' + nndist p p' := by
-  simp only [← NNReal.coe_le_coe, NNReal.coe_add, ← dist_nndist, dist_vadd_vadd_le]
+  simp only [← NNReal.coe_le_coe, NNReal.coe_add, ← dist_nndist]
+  exact dist_vadd_vadd_le v v' p p'
```

Comment:
It's a bit concerning that a lot of `simp`s in this file don't work any more.

Reply:
The whole `vadd` and `vsub` stuff seems rather tedious to port. I can definitively see that there might be ways to improve this part of the library, but it is probably a project for after the port.

Reply:
I think this is a problem with simp itself, not the library